### PR TITLE
Better presubmit instructions for rebasing/merging, #99

### PR DESCRIPTION
**Pull request by advayDev1**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Quick start
+i## Quick start
```
Comment (brunobowden) [inline CONTRIBUTING.md]: Typo inserting "i" at start of header

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -59,21 +59,30 @@ git push
...
 with rebasing and merging, first read:
 https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history 
+
+The following steps will:
+
+1. Update your repository with changes upstream.
+2. Allow you to merge those change in to yours.
+3. Allow you to squash all your commits into a single well-described commit.
+
 ```
```
Comment (brunobowden) [inline CONTRIBUTING.md]: Add notes on merging upstream changes:

git pull

Also, what about configuring upstream if they need to do that

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -59,21 +59,30 @@ git push
...
 git checkout patch-1
 # condense this down to one commit to preserve proper project history
-git rebase -i 5665f0a5ff774005ce30e088ab2e1b76caca5a43
+git rebase -i master
```
Comment (brunobowden) [inline CONTRIBUTING.md]: Do you mean:

    git rebase -i upstream/master

Comment (advayDev1) [inline CONTRIBUTING.md]: No upstream merged into origin in a prior step.

File: CONTRIBUTING.md (hunk 2 above)
Comment (advayDev1) [inline CONTRIBUTING.md]: See L50 through 54, it is up there I think.

**Pull request merged**